identity-and-access-management/secret-manager/how-to/filter-secrets.mdx

```diff
@@ -0,0 +1,58 @@
+---
+meta:
+  title: How to filter your secrets
+  description: Learn how to efficiently filter secrets by name, type, and tags with our detailed documentation, and enhance your security practices.
+content:
+  h1: How to filter your secrets
+  paragraph: Learn how to efficiently filter secrets by name, type, and tags with our detailed documentation, and enhance your security practices.
+tags: sensitive-data filter secret-type
+dates:
+  validation: 2024-07-10
+  posted: 2024-07-10
+categories:
+  - identity-and-access-management
+---
+
+This page explains how to filter Secret Manager secrets by name, [type](/identity-and-access-management/secret-manager/concepts/#secret-types), and tags.
+
+Filtering secrets by name allows you to quickly locate specific secrets, especially in environments with a large number of secrets. This helps in managing and retrieving the required information swiftly.
+
+Filtering secrets by [type](/identity-and-access-management/secret-manager/concepts/#secret-types) allows you to categorize your secrets based on their usage (e.g., API keys, passwords, certificates). This helps in managing different types of secrets effectively.
+
+Filtering by tags allows you to categorize your secrets with more flexibility. Unlike rigid hierarchical structures, tags can be applied dynamically to group secrets based on various criteria, such as Projects or environments.
```

Comment:
We should avoid mentioning the word "projects", it could be confused with Scaleway Projects